config/cl.yml

```diff
@@ -4,10 +4,12 @@ idspace: CL
 base_url: /obo/cl
 
 products:
-- cl.owl: https://raw.githubusercontent.com/obophenotype/cell-ontology/master/cl.owl
-- cl.obo: https://raw.githubusercontent.com/obophenotype/cell-ontology/master/cl.obo
-- cl-basic.owl: https://raw.githubusercontent.com/obophenotype/cell-ontology/master/cl-basic.owl
-- cl-basic.obo: https://raw.githubusercontent.com/obophenotype/cell-ontology/master/cl-basic.obo
+- cl.owl: https://github.com/obophenotype/obophenotype/cell-ontology/releases/latest/download/cl.owl
+- cl.obo: https://github.com/obophenotype/obophenotype/cell-ontology/releases/latest/download/cl.obo
+- cl-basic.owl: https://github.com/obophenotype/obophenotype/cell-ontology/releases/latest/download/cl-basic.owl
+- cl-basic.obo: https://github.com/obophenotype/obophenotype/cell-ontology/releases/latest/download/cl-basic.obo
+- cl-plus.owl: https://github.com/obophenotype/obophenotype/cell-ontology/releases/latest/download/cl-plus.owl
```

Comment:
The `products` list should be limited to `IDSPACE.FORMAT`, so `cl-basic.owl` and `cl-basic.obo` should not be allowed. It looks like they've been in here for 7 years, since the initial migration, so I guess we'll keep that. But I don't want to add any more exceptions, so please remove `cl-plus.owl` and `cl-plus.obo`. They should have `exact` entries below.